protocol/daemons/server/liquidation.go

```diff
@@ -25,6 +26,15 @@ func (server *Server) WithDaemonLiquidationInfo(
 	return server
 }
 
+// SetSubaccountOpenPositions stores the list of subaccount open positions in a go-routine safe map.
+// Placeholder to allow for testing.
+func (s *Server) SetSubaccountOpenPositions(
```

Comment:
@jayy04 I've added this as a placeholder so I can modify this datastructure for testing. I imagine there will be a new function created for this in one of your PRs